website/content/docs/nia/enterprise/index.mdx

```diff
@@ -7,8 +7,21 @@ description: >-
 
 # Consul-Terraform-Sync Enterprise
 
-Consul-Terraform-Sync Enterprise features address organization complexities of collaboration, operations, scale, and governance. Please see the sidebar navigation on the left to view all enterprise features.
+Consul-Terraform-Sync (CTS) Enterprise is available with [Consul Enterprise](https://www.hashicorp.com/products/consul) and requires a Consul [license](/docs/nia/enterprise/license) to be applied.
 
-Enterprise features require a Consul [license](/docs/nia/enterprise/license) to be applied.
+Enterprise features of CTS address organization complexities of collaboration, operations, scale, and governance. CTS Enterprise supports an official integration with [Terraform Cloud](https://www.terraform.io/cloud) to extend insight into dynamic updates of your network infrastructure.
 
-These features are part of Consul-Terraform-Sync Enterprise which is part of [Consul Enterprise](https://www.hashicorp.com/products/consul).
+| Features | Open Source | Enterprise |
+|----------|-------------|------------|
+| | Self-Managed | Self-Managed |
+| Consul Namespace | Default namespace | Filter task triggers by namespace |
+| Network Automation | Terraform OSS | Terraform OSS or Terraform Cloud |
+| Terraform Workspaces | Local | Local or [remote workspaces](https://www.terraform.io/cloud-docs/workspaces) which include run [history](https://www.terraform.io/docs/cloud/run/manage.html), [triggers](https://www.terraform.io/docs/cloud/workspaces/run-triggers.html) and [notifications](https://www.terraform.io/docs/cloud/workspaces/notifications.html) |
```

Comment:
CTE Enterprises support local workspace if the driver is Terraform CLI. It is not clear from the description that the local workspace is only supported in CTS-E if the driver is TF CLI. It would be good to have that clarification